.github/workflows/binaries.yml

```diff
@@ -3,43 +3,32 @@ on:
   release:
     types:
       - created
+permissions:
+  contents: write
 jobs:
   binaries:
-    strategy:
-      matrix:
-        include:
-          - os: ubuntu-latest
-            asset_name: api-linux-amd64.tar.gz
-            compress_cmd: tar -czvf
-            build_cmd: docker run -v $PWD:/data golang:1.19 bash -c "git config --global --add safe.directory /data && cd /data && go build ./cmd/api"
-          - os: windows-latest
-            asset_name: api-windows-amd64.zip
-            compress_cmd: tar.exe -a -c -f
-            build_cmd: go build -o api ./cmd/api
-          - os: macos-latest
-            asset_name: api-darwin-amd64.tar.gz
-            compress_cmd: tar -czvf
-            build_cmd: go build -o api ./cmd/api
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-latest
+    container:
```

Comment:
[goreleaser-cross](https://github.com/goreleaser/goreleaser-cross) builds the binary in this container.